analysis/import/MetricGardenerImporter/build.gradle

```diff
@@ -1,6 +1,5 @@
 dependencies {
     implementation project(':tools:InteractiveParser')
-    implementation project(':import:SourceCodeParser')
```

Comment:
I dont know why MetricGardener was dependent on SourceCodeParser, I removed that. Since nothing broke I assume this was just an unneeded leftover?